packages/hello-gsm-admin/src/components/ListHeader/index.tsx

```diff
@@ -23,19 +31,42 @@ const ListHeader: React.FC<ListHeaderType> = ({
     <S.ListHeaderWrapper>
       <S.TitleWrapper>
         <S.FinalSubmit>최종 제출&nbsp;</S.FinalSubmit>
-        <S.SubmitNumber>{86}</S.SubmitNumber>
+        <S.SubmitNumber>{submitCount}</S.SubmitNumber>
       </S.TitleWrapper>
       <S.ListHeaderContent>
-        <S.SearchInputBox>
-          <S.SearchInput
-            value={searchValue}
-            onFocus={handleFocus}
-            onBlur={handleFocus}
-            placeholder={isFocus ? '' : '검색어를 입력해주세요.'}
-            onChange={e => setSearchValue(e.target.value)}
-          />
-          <SearchIcon />
-        </S.SearchInputBox>
+        <S.ButtonWrapper>
+          <S.SelectWrapper>
+            <S.SelectBox
+              name="tags"
+              id="tag"
+              value={searchTag}
+              onChange={e => {
+                const tagValue = e.target.value;
+                if (
+                  tagValue === 'PHONE_NUMBER' ||
+                  tagValue === 'SCHOOL' ||
+                  tagValue === 'APPLICANT'
+                ) {
+                  setSearchTag(tagValue);
+                }
+              }}
```

Comment:
```suggestion
              onChange={e => {
                const tagValue = e.target.value as SearchTagType;
                  setSearchTag(tagValue);
              }}
```

타입 단언을 사용하면 복잡한 if문을 줄일 수 있을 것 같습니다!

Reply:
타입 단언으로 표시되는게 확실히 깔끔해 보이네요! suggestion감사합니다!

Reply:
들여쓰기가 이상해서 빌드에러가 나네요 ㅠㅠ 다시 알맞게 수정 부탁드립니다..!

Reply:
[04770f7](https://github.com/themoment-team/hello-gsm-front/pull/272/commits/04770f716a3e31d027d0df98e74330d2e805611f)
네 수정 되었습니다!